package.json

```diff
@@ -16,6 +16,7 @@
     "start": "node server.js",
     "clean": "rimraf dist",
     "build": "npm run clean && npm run build:webpack",
+    "build:dev": "webpack --config webpack.config.js",
```

Comment:
If we don't include eslint in the build, it is almost 2x faster.

There is also some way to tell webpack to skip processing some library files, it was drastic improvement but was not working ;) Needed more work.

Also another way is there some parallel build, haven't examined it.

Not sure the compact thing works, maybe I did't configure it correctly, since the speeds look the same, if you can figure out what I did wrong, would be great.